apple/Sources/Sargon/Extensions/Methods/RadixConnect/WalletInteraction/DappToWalletInteractionSubintentExpireAtTime+Wrap+Functions.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import SargonUniFFI
+
+extension DappToWalletInteractionSubintentExpireAtTime {
+	public var date: Date {
+		.init(timeIntervalSince1970: TimeInterval(unixTimestampSeconds))
```

Comment:
It was helpful for iOS, not sure if you need it in Android as well @micbakos-rdx